sql/core/src/test/scala/org/apache/spark/sql/DataFrameSuite.scala

```diff
@@ -572,6 +572,22 @@ class DataFrameSuite extends QueryTest with SharedSQLContext {
     assert(df.schema.map(_.name) === Seq("value"))
   }
 
+  test("drop column using drop with column reference with case-insensitive names") {
+    val col1 = testData("KEY")
```

Comment:
Could you check this on both cases: spark.sql.caseSensitive=true/false?

Reply:
+1 for @maropu 's comment.

Reply:
With that flag set as `true`, an exception was thrown out `org.apache.spark.sql.AnalysisException: Cannot resolve column name "KEY" among (key, value);`. It looks to be the correct behavior in that case. do I still need to check it in this test case and make sure such exception is thrown and caught?

Reply:
Yep. For case-sensitive issue, we need to have a test coverage for both.